examples/miral-shell/miral-xrun.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
-x11_server=Xmir
+x11_server=Xwayland
+xmir_installed=$(which Xmir | wc -l)
+if [ "${xmir_installed}" != "0" ]
```

Comment:
```
$ which Xfoo | wc -l # Xfoo doesn't exist
1
```
This looks wrong, I'd use `if [ -x "$( which Xmir )" ]` instead.

Reply:
Ah it's fine in `bash`, apparently `zsh` is kinda stupid and prints `blah not found` on stderr...
```
$ zsh -c 'which blah'
blah not found
$ bash -c 'which blah'
$
```
Any case, it looks wrong ;)